static/scss/post-list.scss

```diff
@@ -13,3 +13,55 @@
     color: $font-grey;
   }
 }
+
+.post-toprow {
+  min-height: 103px;
+  height: auto;
+  margin-bottom: 10px;
+  overflow: hidden;
+}
+
+.post-toprow-column2 {
```

Comment:
I might suggest nesting `.post-toprow-column2` inside of `.post-toprow` so you can just call the columns something like `col-1` and `col-2` (just for readability)

Reply:
done